tools/DfmHttpService/Model/ServiceContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmHttpService
+{
+    using System.Net;
+
+    public class ServiceContext
```

Comment:
>ServiceContext [](start = 17, length = 14)

how about move out of 'Model'? And this seems more like host instead of context?